extensions/wikia/Blogs/css/oasis.scss

```diff
@@ -261,7 +261,7 @@
 	margin: 4px 2px 0;
 }
 
-wpCategoryInput {
+#wpCategoryInput {
```

Comment:
I couldn't find element with this selector in our repo.
We are styling this element couple of times but I don't see where it is created.